cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("--times", type=int, default=10)
```

Comment:
The '--times' argument could be more descriptive. Consider renaming it to '--retry-times' or '--connection-attempts'.